Header header
geometry_msgs/Point pdes
geometry_msgs/Point vdes
geometry_msgs/Point ades
geometry_msgs/Point ddes
int32 controller 
float32 time
# use integer 0 for position controller and 1 for velocity controller
